README.md

```diff
@@ -48,7 +62,8 @@ cmake .. -DCMAKE_INSTALL_PREFIX=$SHADROOT \
          -DSHAD_RUNTIME_SYSTEM=TBB        \
          -DTBB_ROOT=$TBBROOT              \
          -DGTEST_ROOT=$GTESTROOT          \
-         -DGPERFTOOLS_ROOT=$GPERFTOOLSROOT
+         -DGPERFTOOLS_ROOT=$GPERFTOOLSROOT \
```

Comment:
Can you please align all the "\" characters?

Sorry, I am kind of affected by OCD :-P